### PR TITLE
update table

### PR DESCRIPTION
**Pull request by tommydcjung**

### CODE REVIEW

**Review by taylor-bsg — COMMENTED** [review]
Rearrange to match original code -- hard to tell what is going on from the diff

File: gcc/config/riscv/bsg_vanilla.md [hunk 1]
```diff
@@ -22,76 +22,82 @@
...
+(define_insn_reservation "bsg_vanilla_imuldiv" 33
   (and (eq_attr "tune" "bsg_vanilla")
-       (eq_attr "type" "imul"))
-  "bsg_vanilla_imuldiv*10")
+       (eq_attr "type" "imul,idiv"))
+  "bsg_vanilla_alu*33")
 
-(define_insn_reservation "bsg_vanilla_idivsi" 34
+;; i2f 
+(define_insn_reservation "bsg_vanilla_i2f" 5
```
Comment (taylor-bsg) [inline gcc/config/riscv/bsg_vanilla.md]: maybe a comment that explains why 5

File: gcc/config/riscv/bsg_vanilla.md [hunk 2]
```diff
@@ -22,76 +22,82 @@
...
-       (and (eq_attr "type" "fadd,fmul,fmadd")
-	    (eq_attr "mode" "SF")))
+       (eq_attr "type" "mfc"))
   "bsg_vanilla_alu")
 
-(define_insn_reservation "bsg_vanilla_fmul_double" 7
+;; fpu_float
+;; note: WAW hazards for FPU not currently modeled.
+;; mostly it applies after branch.
+(define_insn_reservation "bsg_vanilla_fpu_float" 5
```
Comment (taylor-bsg) [inline gcc/config/riscv/bsg_vanilla.md]: maybe a comment that explains why 5? Isn't it 3?

Comment (tommydcjung) [reply gcc/config/riscv/bsg_vanilla.md]: For the old pipeline, FP_EXE -> F -> P -> U -> FP_WB. It's total of 5 cycles.

**Comment by tommydcjung** [comment]
> Add a comment here:
> 
> // see code gen pattern match rules in riscv.md
> // for what is considered to be fmove
> // fabs, fsgnj, fneg, fmin, fmax, float, riscv_output_move(float,float)->fmv.s
> 
> In terms of the latency are all of those 3 cycle?

Yes, i->f move, sign inject, i2f, fma are all 3 cycles.

**Review by drichmond — APPROVED** [review]

**Pull request merged**